modules/tutorials/pages/pattern-matching/one-hop-patterns.adoc

```diff
@@ -13,37 +13,43 @@ Currently, pattern matching may only be used in read-only queries. DML support w
 
 [NOTE]
 ====
-Pattern matching queries support nested queries
+Pattern matching queries support nested queries.
 ====
 
 == 1-Hop Pattern
 
 The easiest way to understand patterns is to start with a simple 1-Hop pattern. Even a single hop has several options. After we've tackled single hops, then we'll see how to add repetition to make variable length patterns and how to connect single hops to form bigger patterns.
 
-In classic GSQL queries, described in xref:tutorials:gsql-101/README.adoc[GSQL 101], we used the punctuation `+-( )->+`  in the FROM clause to indicate a 1-hop query, where the arrow specifies the vertex flow from left to right, and `( )` encloses the edge types.
+In classic GSQL queries, described in xref:tutorials:gsql-101/README.adoc[GSQL 101], we used the punctuation `+-( )->+`  in the FROM clause to indicate a 1-hop query.
 
+There the arrow `+->+` specifies the vertex flow from left to right and the parentheses `( )` enclose the edge types.
+
+.Classic GSQL Example
 [source,gsql]
 ----
-Person:p -(LIKES:e)-> Message:m          /* Classic GSQL example */
+Person:p -(LIKES:e)-> Message:m
```

Comment:
This is a good example of good and consistent use of empty space in a path pattern. In template form:
`V1:v1 -(E1:e1)- V2:v2 -(E2:e2)- V3:v3`
I think the dashes - should be directly adjacent to the parentheses: -(edge)-
I prefer to have a space separating the dash from the vertex. This space could be eliminated, but the same rule should be applied for the entire path. Here is an ugly example of bad, inconsistent spacing, on the multiple-hop-and-accumulation page:
`Person:s-(LIKES>) -:msg - (HAS_CREATOR>)-Person:t`
Can you check FROM clauses and improve the path pattern spacing?